### ISSUE TITLE
Should allow DateTimeString of 610 in OccurrenceDateTime in xslt/ZF_232/FACTUR-X_EN16931.xslt

### ISSUE THREAD

**Issue by jmichler** [ZUGFeRD/mustangproject#604]
In our ZUGFeRD Invoice we have:
        <ram:ApplicableHeaderTradeDelivery>
            <ram:ActualDeliverySupplyChainEvent>
                <ram:OccurrenceDateTime>
                    <udt:DateTimeString format="610">202403</udt:DateTimeString>
                </ram:OccurrenceDateTime>
            </ram:ActualDeliverySupplyChainEvent>
        </ram:ApplicableHeaderTradeDelivery>
We run:
java -jar ~/Desktop/zugferd-validierung/files3/Mustang-CLI-2.15.0.jar --action validate --source Beleg.pdf

This gives us:
<error type="4" location="/*:CrossIndustryInvoice[namespace-uri()='urn:un:unece:uncefact:data:standard:CrossIndustryInvoice:100'][1]/*:SupplyChainTradeTransaction[namespace-uri()='urn:un:unece:uncefact:data:standard:CrossIndustryInvoice:100'][1]/*:ApplicableHeaderTradeDelivery[namespace-uri()='urn:un:unece:uncefact:data:standard:ReusableAggregateBusinessInformationEntity:100'][1]/*:ActualDeliverySupplyChainEvent[namespace-uri()='urn:un:unece:uncefact:data:standard:ReusableAggregateBusinessInformationEntity:100'][1]/*:OccurrenceDateTime[namespace-uri()='urn:un:unece:uncefact:data:standard:ReusableAggregateBusinessInformationEntity:100'][1]/*:DateTimeString[namespace-uri()='urn:un:unece:uncefact:data:standard:UnqualifiedDataType:100'][1]" criterion="document('FACTUR-X_EN16931_codedb.xml')//cl[@id=3]/enumeration[@value=$codeValue3]">Value of '@format' is not allowed. [ID FX-SCH-A-000022] from /xslt/ZF_232/FACTUR-X_EN16931.xslt)</error>

This is a bug though I think, see 6.5.3 of https://konik.io/ZUGFeRD-Spezifikation/Das-ZUGFeRD-Format_1p0.pdf which allows this format 610 (=the entire month) beside of specifying an individual day.

**Comment by jmichler** [comment]
In generating this is supported; see https://github.com/ZUGFeRD/mustangproject/issues/136

**Comment by langfr** [comment]
FACTUR-X_EN16931.xslt not an artifact created by the mustangproject but is a part of the ZUGFeRD 2.3.2 specification, downloadable from https://www.ferd-net.de/standards/zugferd-2.3.2/zugferd-2.3.2.html e.g.
So in my opinion it's no good idea to modify this.

**Comment by jmichler** [comment]
I tend to agree; this seems to be a bit inconsistent though. Do you know if there is a forum to discuss this on the ferd-net.de side?

**Comment by jstaerk** [comment]
Yes there is a forum and a gitlab, unfortunately only for volunteers -> I guess the official answer would point somewhere in the direction of https://www.ferd-net.de/ueber-uns/ferd-und-competence-center/mitwirken-im-ferd
